cpp/ql/test/library-tests/ir/ir/raw_ir.expected

```diff
@@ -4366,3 +4366,55 @@ ir.cpp:
 #  987|     v0_22(void)             = UnmodeledUse             : mu*
 #  987|     v0_23(void)             = ExitFunction             : 
+
+#  991| int AsmStmt(int)
+#  991|   Block 0
+#  991|     v0_0(void)        = EnterFunction            : 
+#  991|     mu0_1(unknown)    = AliasedDefinition        : 
+#  991|     mu0_2(unknown)    = UnmodeledDefinition      : 
+#  991|     r0_3(glval<int>)  = VariableAddress[x]       : 
+#  991|     mu0_4(int)        = InitializeParameter[x]   : r0_3
+#  992|     mu0_5(unknown)    = InlineAsm                : mu0_2
+#  993|     r0_6(glval<int>)  = VariableAddress[#return] : 
+#  993|     r0_7(glval<int>)  = VariableAddress[x]       : 
+#  993|     r0_8(int)         = Load                     : r0_7, mu0_2
+#  993|     mu0_9(int)        = Store                    : r0_6, r0_8
+#  991|     r0_10(glval<int>) = VariableAddress[#return] : 
+#  991|     v0_11(void)       = ReturnValue              : r0_10, mu0_2
+#  991|     v0_12(void)       = UnmodeledUse             : mu*
+#  991|     v0_13(void)       = ExitFunction             : 
+
+#  996| void AsmStmtWithOutputs(unsigned int&, unsigned int&, unsigned int&, unsigned int&)
+#  996|   Block 0
+#  996|     v0_0(void)                  = EnterFunction          : 
+#  996|     mu0_1(unknown)              = AliasedDefinition      : 
+#  996|     mu0_2(unknown)              = UnmodeledDefinition    : 
+#  996|     r0_3(glval<unsigned int &>) = VariableAddress[a]     : 
+#  996|     mu0_4(unsigned int &)       = InitializeParameter[a] : r0_3
+#  996|     r0_5(glval<unsigned int &>) = VariableAddress[b]     : 
+#  996|     mu0_6(unsigned int &)       = InitializeParameter[b] : r0_5
+#  996|     r0_7(glval<unsigned int &>) = VariableAddress[c]     : 
+#  996|     mu0_8(unsigned int &)       = InitializeParameter[c] : r0_7
+#  996|     r0_9(glval<unsigned int &>) = VariableAddress[d]     : 
+#  996|     mu0_10(unsigned int &)      = InitializeParameter[d] : r0_9
+#  999|     mu0_11(unknown)             = InlineAsm              : mu0_2
+# 1010|     v0_12(void)                 = NoOp                   : 
+#  996|     v0_13(void)                 = ReturnVoid             : 
+#  996|     v0_14(void)                 = UnmodeledUse           : mu*
+#  996|     v0_15(void)                 = ExitFunction           : 
+
+# 1002|   Block 1
+# 1002|     r1_0(glval<unsigned int &>) = VariableAddress[a] : 
+# 1002|     r1_1(unsigned int &)        = Load               : r1_0, mu0_2
+
+# 1002|   Block 1
+# 1002|     r1_0(glval<unsigned int &>) = VariableAddress[b] : 
+# 1002|     r1_1(unsigned int &)        = Load               : r1_0, mu0_2
+
+# 1002|   Block 1
+# 1002|     r1_0(glval<unsigned int &>) = VariableAddress[c] : 
+# 1002|     r1_1(unsigned int &)        = Load               : r1_0, mu0_2
+
+# 1002|   Block 1
+# 1002|     r1_0(glval<unsigned int &>) = VariableAddress[d] : 
+# 1002|     r1_1(unsigned int &)        = Load               : r1_0, mu0_2
```

Comment:
There are four blocks named "Block 1". Is that supposed to be possible? If the numbering is based on a depth-first search, then I guess it's plausible that unreachable blocks won't have unique numbers.